src/components/ValueCard/ValueCard.story.jsx

```diff
@@ -38,6 +38,37 @@ storiesOf('Watson IoT/ValueCard', module)
       </div>
     );
   })
+  .add('small / with expand', () => {
+    const size = select('size', Object.keys(CARD_SIZES), CARD_SIZES.SMALL);
+    return (
+      <div
+        style={{
+          width: text('cardWidth', `${getCardMinSize('lg', size).x}px`),
+          margin: spacing05 + 4,
+        }}
+      >
+        <ValueCard
+          title={text('title', 'Occupancy')}
+          id="facilitycard"
+          content={{
+            attributes: object('attributes', [
+              {
+                dataSourceId: 'occupancy',
+                unit: '%',
+              },
+            ]),
+          }}
+          availableActions={{
+            expand: true,
+          }}
+          breakpoint="lg"
+          size={size}
+          values={{ occupancy: number('occupancy', 88) }}
+          onCardAction={() => action('Value Card Expand Action')}
```

Comment:
Clicking the expand button causes an error
![image](https://user-images.githubusercontent.com/5167091/93208350-b8efda00-f75c-11ea-89f4-ad5acb3bc070.png)

Reply:
Fixed